### DISCUSSION TITLE
Daily average statistics for received / send

### DISCUSSION THREAD

**Discussion by hurvajs** [devmount/third-stats#404] [Ideas]
Thank you for the great app
nevertheless it can be improved - as I need to check what it an average  ammount of received/sent emails from certain period

 At the moment I can only see the Mails per day (probably together receive+sent) but activity of an user is indicated by sent emails quantity not received and I need to learn how the email user is active.

**Comment by devmount** [comment]
Thank you for this suggestion. I see two possible solutions here:

## 1. Making total/received/sent email numbers togglable separatly

The "Mails per _unit_" featured number is (as you already guessed) simply the total number of emails averaged down to the corresponding unit. This could be removed and instead, the total / received / sent featured numbers could be made configurable for showing the average per unit. Here is a possible concept:

![image](https://user-images.githubusercontent.com/5441654/228045326-2074fd60-4e1a-4923-9282-14032c1782e1.png)

## 2. Providing one single toggle bar for all featured numbers

This is similar to solution 1, with the difference of only having one single toggle bar for all suitable featured numbers simultaniously. Possible concept:

![image](https://user-images.githubusercontent.com/5441654/228046789-49dd4255-980e-4828-b996-de57deefb31a.png)

Both solutions have the advantage of reducing the featured numbers sections, making it more clear, which numbers are shown (total, received or sent) while giving the opportunity to show averaged numbers for all.
